File: tusow/src/main/java/it/unibo/coordination/tusow/exceptions/ServerError.java
package it.unibo.coordination.tusow.exceptions;

public class ServerError extends HttpError {

    public ServerError(int statusCode) {
        super(statusCode);
    }

    public ServerError(int statusCode, String message) {
        super(statusCode, message);
    }

    public ServerError(int statusCode, String message, Throwable cause) {
        super(statusCode, message, cause);
    }
}

File: tusow-service/src/main/kotlin/it/unibo/coordination/tusow/exceptions/ServerError.kt
package it.unibo.coordination.tusow.exceptions

open class ServerError : HttpError {
    constructor(statusCode: Int) : super(statusCode)
    constructor(statusCode: Int, message: String?) : super(statusCode, message)
    constructor(statusCode: Int, message: String?, cause: Throwable?) : super(statusCode, message, cause)
}